json-rpc/eth_call.md

```diff
@@ -0,0 +1,113 @@
+
+
+# `eth_call`
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+Specification | Description 
+--|--
+1         | `eth_call` MUST NOT affect on-chain state|
+2         | `eth_call` MUST NOT accept requests that are not in valid JSON format
+3         | `eth_call` MUST return a response in valid JSON format
+4         | `eth_call` MUST accept a `params` parameter [JSON Array](https://tools.ietf.org/html/rfc7159#section-5) of length 2, and its first item containing a [JSON Object](https://tools.ietf.org/html/rfc7159#section-4) with keys known here as the `input parameters`
+4.1       | `eth_call` MUST accept an input parameter `from` of type `DATA` [!]
```

Comment:
It appears the specification is worded with a target audience of server developers in mind.  This API is bi-directional and this document is likely to be read by both client developers and server developers.  In this case, the `from` property is optional but this line makes it sound like it is required if you are reading from the point of view of a client.  Consider trying to phrase things in this document in a way that is neutral to whether the reader is a server or client developer.

> The `from` property is optional, but if present its type **MUST** be 20 byte DATA.